tests/examples/css-variables.css

```diff
@@ -0,0 +1,8 @@
+:root {
```

Comment:
this is what they use in bootstrap

Reply:
Perhaps that should be in a CSS comment right before. So we don't forget.